package.json

```diff
@@ -14,7 +14,7 @@
   },
   "license": "Apache-2.0",
   "engines": {
-    "node": ">= 6"
+    "node": ">= 16"
```

Comment:
How and where is this library consumed and used? Does bumping this up so heavily cause any issues?

Reply:
It would, but http2 support was only introduced in node 15, and node 16 is the lowest LTS version that supports http2 hence decided/forced to make this change